connectors/migrations/db/migration_31.sql

```diff
@@ -0,0 +1,16 @@
+-- Migration created on Nov 07, 2024
```

Comment:
I thought I had already changed the types there, did you run `migration_30` before running `create-db-migration` @albandum?

Reply:
Yeah I did apply, but the create-migration script added a bunch of stuff that I already removed. Will remove the type switching